### PR TITLE
better mcse_sd

### PR DESCRIPTION
**Pull request by avehtari**
#### Summary

PR for issue #232 better mcse_sd

#### Copyright and Licensing

By submitting this pull request, the copyright holder is agreeing to 
license the submitted work under the following licenses:

- Code: BSD 3-clause (https://opensource.org/licenses/BSD-3-Clause)
- Documentation: CC-BY 4.0 (https://creativecommons.org/licenses/by/4.0/)

### CODE REVIEW

File: R/convergence.R [hunk 1]
```diff
@@ -322,41 +321,6 @@ ess_mean.rvar <- function(x, ...) {
   summarise_rvar_by_element_with_chains(x, ess_mean, ...)
 }
 
-#' Effective sample size for the standard deviation
```
Comment (paul-buerkner) [inline R/convergence.R:325]: You seem to have removed the `ess_sd` function. Was that on purpose? We cannot just remove previously exported functions even if we don't use them anymore in the mcse computation. 

I suggest we add them back and only change the `mcse_sd` function.

**Review by paul-buerkner — CHANGES_REQUESTED** [review]

File: R/convergence.R (hunk 1 above)
Comment (avehtari) [reply R/convergence.R:325]: I put it back with the formula used in mcse_sd

Comment (paul-buerkner) [reply R/convergence.R:325]: Thank you!

**Comment by paul-buerkner** [comment]
I will make some more manual checks and perhaps fix tests before merging but otherwise looks good.

**Comment by paul-buerkner** [comment]
Merging now after a few minor changes.

**Review by paul-buerkner — APPROVED** [review]

**Pull request merged**